services/121-service/src/shared/services/custom-http.service.ts

```diff
@@ -274,7 +274,12 @@ export class CustomHttpService {
     }
 
     // Explicitly mask the username/email:
-    if (redactedData.username) {
+    if (
+      redactedData &&
+      typeof redactedData === 'object' &&
```

Comment:
Ideally, this would be inferred automatically with the `isPlainObject` check, but that function would have to return some information that would help TS infer it (which it currently doesn't).

Reply:
Haha, my whole point of using that Lodash-helper was to avoid these verbose type-check-syntax-things.. :)

There are types installed for Lodash (using `@types/lodash`), but apparently that doesn't do the trick?
There will probably be some (TS-specific/native) alternative to the `isPlainObject`-check, right?

Reply:
> There are types installed for Lodash (using @types/lodash), but apparently that doesn't do the trick?

Nope, the [lodash function only returns a boolean](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/880e19374f4034deeacd43785eac84d52b197e00/types/lodash/common/lang.d.ts#L1115-L1138) rather than [using a type-guard](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates).

I assume that is the case also because 

> There will probably be some (TS-specific/native) alternative to the isPlainObject-check, right?

Maybe `typeof data === 'object'`? But I'm not entirely sure what we're trying to achieve/avoid.